tests/e2e/tests/pilot/testdata/authn/v1alpha1/authn-policy-jwt.yaml.tmpl

```diff
@@ -0,0 +1,15 @@
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "enable-jwt"
+spec:
+  targets:
+  - name: "c"
+  - name: "d"
+    ports:
+    - number: 80
+  origins:
+  - jwt:
+      issuer: "628645741881-noabiu23f5a8m8ovd8ucv698lj78vv0l@developer.gserviceaccount.com"
+      jwksUri: "https://www.googleapis.com/service_accounts/v1/jwk/628645741881-noabiu23f5a8m8ovd8ucv698lj78vv0l@developer.gserviceaccount.com"
```

Comment:
What is this account? Could it go away?

Reply:
This service account is under GKE project 628645741881(mwan-endpoints2), I added comment that test will fail if this service account is deleted, since proxy uses it in sample, I assume it's stable.

Reply:
Istio proxy and https://github.com/cloudendpoints/esp use this project for test as well.

Reply:
I would argue that proxy and endpoint may not be the best model :) (not to mention the jwt code in proxy code is going to move, so you may want to keep your comments refer to ESP instead). Anyway, I am ok with this for now, and we can think to improve it later.